### PR TITLE
Problem: not updated for version 0.3.1 (fix #719)

### LINKED ISSUE #719: Problem: not updated for version 0.3.1
update to 0.3.1

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: update to 0.3.1

### CODE REVIEW

**Review by tomtau — CHANGES_REQUESTED** [review]
maybe you can update the other Cargo.toml and the changelog?

**Comment by leejw51crypto** [comment]
ok

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Changelog
-## [0.x.x] - unreleased
+## [0.3.1] - 2022-12-6
 ### Added
 - Dynamic Contract C++ Bindings 
 - Minting C++ Example
```
Comment (tomtau) [inline CHANGELOG.md:5]: is the changelog complete?

**Review by tomtau — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (leejw51crypto) [reply CHANGELOG.md:5]: i'll add

**Comment by damoncro** [comment]
Please update Cargo.lock as well.

**Comment by leejw51crypto** [comment]
ok

**Review by SHH-HaoZhang — APPROVED** [review]

**Pull request merged**